fish-rust/src/wchar_ext.rs

```diff
@@ -88,6 +88,11 @@ pub trait WExt {
             self.as_char_slice().iter().copied().rev(),
         )
     }
+
+    /// Equivalent of `basic_string.rfind()` from C++
+    fn find_last_char(&self, c: char) -> Option<usize> {
```

Comment:
How about calling it `rfind`, like the [method on `str`](https://doc.rust-lang.org/std/primitive.str.html#method.rfind)?

Reply:
Sounds good, I wasn't sure whether to name it the same or name it like the `find_char` method in the same trait extension.